WeatherPlannerDemoTests/Sourcery/Generated/AutoMockable.generated.swift

```diff
@@ -0,0 +1,61 @@
+// Generated using Sourcery 1.0.2 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+
+// swiftlint:disable variable_name
+// swiftlint:disable line_length
+
+import RxSwift
+@testable import WeatherPlannerDemo
+class CityDetailsUseCaseProtocolMock: CityDetailsUseCaseProtocol {
```

Comment:
Where did you use these mock classes?

Reply:
I have a really unique case where I have to create CoreData objects with context, so auto mocked classes do not work for me, however in terms of practice I have left them in project, until I figure out a better use for them